### PR TITLE
feat: add um-idcard

### PR DESCRIPTION
**Pull request by mockdot**
## Description

- Add um-idcard

## Checklist

- [ ] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My code fits the style guidelines.
- [x] My PR fits the contribution guidelines.

### CODE REVIEW

File: server/main.lua [hunk 1]
```diff
@@ -11,24 +11,12 @@ local availableJobs = {
 -- Functions
 
 local function giveStarterItems()
-    local Player = QBCore.Functions.GetPlayer(source)
+    local src = source
```
Comment (Manason) [inline server/main.lua]: Where does source come from? Need to pass it as a parameter

**Review by Manason — CHANGES_REQUESTED** [review]

**Review by Manason — APPROVED** [review]

File: client/main.lua [hunk 2]
```diff
@@ -315,8 +319,10 @@ RegisterNetEvent('QBCore:Client:OnPlayerUnload', function()
     deletePeds()
 end)
 
-RegisterNetEvent('qb-cityhall:client:getIds', function()
-    TriggerServerEvent('qb-cityhall:server:getIDs')
+RegisterNetEvent('qbx-core:server:createCharacter', function()
```
Comment (Manason) [inline client/main.lua]: Isn't this a server side event? Also server would never have source since it isn't coming from a player

**Review by Manason — CHANGES_REQUESTED** [review]

**Comment by mockdot** [comment]
Looks like server side does not like using "source" even if it is a function, changed it to src and it now works as expected.

File: server/main.lua [hunk 3]
```diff
@@ -143,7 +113,11 @@ RegisterNetEvent('qb-cityhall:server:ApplyJob', function(job)
     TriggerClientEvent('QBCore:Notify', src, Lang:t('info.new_job', {job = JobInfo.label}), 'success')
 end)
 
-RegisterNetEvent('qb-cityhall:server:getIDs', giveStarterItems)
+RegisterNetEvent('qbx-core:server:createCharacter', function()
+    local src = source
+    if not src then return end
```
Comment (Manason) [inline server/main.lua]: net event will always have source if called from the client, no need to check for its existence.

Also secure this net event please :)
https://manason.github.io/effective-fivem-lua/events/#secure-net-events

**Review by Manason — CHANGES_REQUESTED** [review]

File: server/main.lua (hunk 3 above)
Comment (mockdot) [reply server/main.lua]: Sounds good, thanks!

**Review by Manason — APPROVED** [review]

**Review by Manason — APPROVED** [review]

**Review by BerkieBb — APPROVED** [review]

**Comment by Manason** [comment]
modify to call qbx_idcard

**Pull request closed without merging**